### PR TITLE
docs: add changeset generate command and version bump type explanations

### PR DESCRIPTION
**Pull request by ComfortablyCoding**
## Scope

What's changed:

- Added changeset generate command
- Added notes on the possible bump versions that can be selected and what types of changes correspond to which bump version.

## Potential Risks / Drawbacks

- None

## Review Notes / Questions

- Is this is better suited to be mentioned in the `Running Dev Environment` page?
- I was not sure what major/minor/patch corresponds to in the context of `directus`. I took the definitions from the [semver equivalents](https://semver.org/).

### CODE REVIEW

**Comment by ComfortablyCoding** [comment]
I already signed the CLA in PR https://github.com/directus/directus/pull/20238 (yet to be merged). Do I sign it again in this one in case it gets merged first?

**Review by rijkvanzanten — APPROVED** [review]

File: docs/contributing/pull-request-process.md [hunk 1]
```diff
@@ -43,3 +43,17 @@ To properly generate changelogs and determine the right version number after a c
...
+
+A changeset can be generated via the following command
+
+```bash
+pnpm changeset
+```
+
+### Changeset bump definitions
+
+The following are the types of version bumps that can be specified in a changeset.
```
Comment (paescuj) [inline docs/contributing/pull-request-process.md:55]:
```suggestion
A changeset can be generated via the following command:

```shell
pnpm changeset
```

### Changeset Bump Definitions

The following are the types of version bumps that can be specified in a changeset:
```

File: docs/contributing/pull-request-process.md [hunk 2]
```diff
@@ -43,3 +43,17 @@ To properly generate changelogs and determine the right version number after a c
 [changesets](https://github.com/changesets/changesets). Each pull request should include a changeset that describes
 whether the change is a patch/minor/major version bump, and describe what the change is. Changesets should be written in
 past tense.
+
+A changeset can be generated via the following command
```
Comment (paescuj) [inline docs/contributing/pull-request-process.md:47]:
```suggestion
A changeset can be generated via the following command:
```

File: docs/contributing/pull-request-process.md [hunk 3]
```diff
@@ -43,3 +43,17 @@ To properly generate changelogs and determine the right version number after a c
...
+
+```bash
```
Comment (paescuj) [inline docs/contributing/pull-request-process.md:49]:
```suggestion
```shell
```

File: docs/contributing/pull-request-process.md (hunk 1 above)
Comment (paescuj) [inline docs/contributing/pull-request-process.md:55]:
```suggestion
### Changeset Bump Definitions

The following are the types of version bumps that can be specified in a changeset:
```

**Comment by paescuj** [comment]
Thanks @ComfortablyCoding ❤️ 

> I already signed the CLA in PR https://github.com/directus/directus/pull/20238 (yet to be merged). Do I sign it again in this one in case it gets merged first?

I've 🍒-picked your CLA signing commit into this PR 👍

**Comment by ComfortablyCoding** [comment]
Thanks!

**Pull request merged**